### PR TITLE
Added admin link to the breadcrumb bar

### PR DESCRIPTION
**Pull request by dtebo**
Updated the breadcrumb bar with an admin link to reflect when the user is on the admin page and also allow the user to easily navigate to the admin page.

### CODE REVIEW

File: src/components/Admin/Admin.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+              <Tag color={color} key={tag}>
+                {tag.toUpperCase()}
+              </Tag>
+            );
+          })}
+        </>
+      ),
+    },
+    {
+      title: 'Action',
```
Comment (Sonyei) [inline src/components/Admin/Admin.js:60]: Looking over the wireframe, I think the only action that should be visible from the landing page is 'edit'. Deactivate/activate should be nested inside the accordion, beneath the form, and on a toggle.

File: src/components/Admin/Admin.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      tags: ['mentor'],
+    },
+];
+
+/*End Placeholder Data*/
+
+const Admin = props => {
+    return(
+        <>
+            <Table columns={columns} dataSource={data} />
```
Comment (Sonyei) [inline src/components/Admin/Admin.js:103]: To keep the UI clean and a little more mobile-friendly, I think we should keep the data listed on the landing page restricted to first & last name, along with the tags.

**Review by Sonyei — COMMENTED** [review]
I left some in-line comments to go over, but overall I think this is a great start. There's just a little too much information displayed on the landing page as per the stakeholder meeting yesterday & our wireframe; I think we should be selective about what data we display and wait for the user to choose to display additional information with the accordion.

File: src/components/Admin/Admin.js (hunk 1 above)
Comment (dtebo) [reply src/components/Admin/Admin.js:60]: Ok, I will go ahead and remove the deactivate link and add the toggle inside the form. I will let you know when that is completed.

File: src/components/Admin/Admin.js (hunk 2 above)
Comment (dtebo) [reply src/components/Admin/Admin.js:103]: That makes sense. I will go ahead and remove the address column. That was actually a placeholder column that went along with the test data that I found on ant design's site anyway.

**Comment by dtebo** [comment]
> I left some in-line comments to go over, but overall I think this is a great start. There's just a little too much information displayed on the landing page as per the stakeholder meeting yesterday & our wireframe; I think we should be selective about what data we display and wait for the user to choose to display additional information with the accordion.

Thank you! I will go ahead and minimize the data that is displayed initially. No problem.

**Review by AngelikaMB — APPROVED** [review]
The implementation of your revisions looks great! Everything looks up to par with our meetings and planning boards.

**Comment by dtebo** [comment]
> The implementation of your revisions looks great! Everything looks up to par with our meetings and planning boards.

Thank you! Much appreciated. Thank you for your feedback on changes to this feature as well. We are making great progress here!

**Pull request closed without merging**